const myName = 'ayon'

try {
    myName = 'huh';
}
catch (error) {
    console.log('getting error', error);

}

console.log(100);
console.log(myName);